force-app/main/default/classes/OpportunityTriggerHelper.cls

```diff
@@ -0,0 +1,175 @@
+public with sharing class OpportunityTriggerHelper {
+    public static boolean isTriggerRunning = false;
+
+    public static void setType(List<Opportunity> opportunities) {
+    for (Opportunity opp : opportunities) {
+        if (opp.Type == null){
+            opp.Type = 'New Customer';
+        }
+    }
+}
+    public static void validateCloseOpportunity(List<Opportunity> opportunities) {
+        for (Opportunity opp : opportunities) {
+                if (opp.IsClosed) {
+                    opp.addError('Cannot delete closed opportunity');
```

Comment:
Try moving the string values into a variable at the top level of the class for easier use.

Reply:
I wasn't exactly sure what you meant here, but I created string variables for the different error messages.